pkg/compute/capacity/system/provider.go

```diff
@@ -54,6 +54,26 @@ func (p *PhysicalCapacityProvider) GetAvailableCapacity(ctx context.Context) (mo
 	}, nil
 }
 
+func (p *PhysicalCapacityProvider) GetTotalCapacity() (models.Resources, error) {
```

Comment:
The other method `GetAvailableCapacity` in this file is applying 80% to all values. I needed something that returned 100% to do maths right.

Reply:
Could we not supply the multiplier to GetAvailableCapacity? So that current calls will pass `0.8` (hopefully a well named constant so we know why 0.8, and this PR can pass 1.0?

Reply:
Yeah, I'd actually made that change initially using options but decided against it. I don't know what 80% of a GPU is, for example. In my opinion `PhysicalCapacityProvider` should only ever provide values representing 100% of the system, and allow callers to make their own decision. But, I don't want to unwind that all here, nor do I know if my opinion would be accepted - thus a new method. 

Care if I just leave this new method as is?